downstairs/src/region.rs

```diff
@@ -1057,15 +1165,47 @@ impl Extent {
         }
         tx.commit()?;
 
+        // Buffer writes for fewer syscalls. The 65536 buffer size here is
+        // chosen somewhat arbitrarily.
+        let mut write_buffer = [0u8; 65536];
```

Comment:
looking for feedback, is this a reasonable size to be allocating on the stack here?

Reply:
I think it's reasonable, yeah. I'd be curious what size OSes submit on average. The f3 test submits something like 80k sized writes, but even buffering at 64k vastly reduces the syscalls there.

I also suspect this relates to https://github.com/oxidecomputer/crucible/issues/101

Reply:
As I pressed submit, I realized that you're asking about stack allocation, not how it relates to the writes we receive from the guest. Disregard previous :)

I don't know about if 64k is reasonable on the stack. I'd say if it works then :shipit: